### PR TITLE
Minor update

### PR DESCRIPTION
**Pull request by Sunaries**
- Properly named the amulets with help of @lordseph.
- Minor fixes.

### CODE REVIEW

**Comment by Sunaries** [comment]
Good to merge.

File: EnglishTranslation-master/ETC.tsv [hunk 1]
```diff
@@ -21292,15 +21292,15 @@ ETC_20160323_021291	Recipe - Vista Cannon
...
-ETC_20160323_021296	Recipe - Agalas Amulet
-ETC_20160323_021297	Recipe - Perejimas Amulet
-ETC_20160323_021298	Recipe - Zibelis Amulet
-ETC_20160323_021299	Recipe - Balvaras Amulet
-ETC_20160323_021300	Recipe - Magiya Amulet
-ETC_20160323_021301	Recipe - Vinis Amulet
+ETC_20160323_021295	Recipe - Demono Amulet
+ETC_20160323_021296	Recipe - Augalo Amulet
+ETC_20160323_021297	Recipe - Perejimo Amulet
+ETC_20160323_021298	Recipe - Zveris Amulet
```
Comment (tosk91) [inline EnglishTranslation-master/ETC.tsv:21298]: Zveries

File: EnglishTranslation-master/ITEM.tsv [hunk 2]
```diff
@@ -10357,24 +10357,24 @@ ITEM_20160323_010356	Can only be used on equipment with 0 Potential. The equipme
...
-ITEM_20160323_010371	[Equip Effect] 5% chance to increase magic amplification by 152 for 30 seconds upon a critical
-ITEM_20160323_010372	Vinis Amulet
-ITEM_20160323_010373	[Equip Effect] 15% chance to inflict [Bleeding] for 3 seconds when using [Slash]-type attacks
+ITEM_20160323_010360	Demono Amulet
+ITEM_20160323_010361	[Effect] 0.8% chance to inflict 8640 additional damage against Devil-type Targets
+ITEM_20160323_010362	Augalo Amulet
+ITEM_20160323_010363	[Effect] 0.8% chance to inflict 8640 additional damage against Plant-type Targets
+ITEM_20160323_010364	Perejimo Amulet
+ITEM_20160323_010365	[Effect] 0.8% chance to inflict 8640 additional damage against Mutant-type Targets
+ITEM_20160323_010366	Zveris Amulet
```
Comment (tosk91) [inline EnglishTranslation-master/ITEM.tsv]: Zveries

**Comment by Sunaries** [comment]
Good to merge.

**Pull request merged**